api/controllers/employees.js

```diff
@@ -0,0 +1,86 @@
+const sequelize = require('sequelize');
+
+const logger = require('../logger');
+const Employee = require('../models/employee');
+const Transaction = require('../models/transaction');
+
+const EmployeeController = {};
+
+EmployeeController.getEmployees = async (req, res, next) => {
+    try {
```

Comment:
let's thing of a nice way of moving `catch` functionality to a wrapper helper.

one example can be https://itnext.io/using-async-await-to-write-cleaner-route-handlers-7fc1d91b220b